src/app/client/src/app/modules/core/services/search/search.service.ts

```diff
@@ -378,37 +379,61 @@ export class SearchService {
   }
 
   getFilterValues(contents) {
-    let subjects = _.map(contents, content => {
-      return (_.get(content, 'subject'));
-    });
-    subjects = _.values(_.groupBy(subjects)).map((subject) => {
+      let subjects = _.map(contents, content => {
+        return (_.get(content, 'subject'));
+      });
+      subjects = _.values(_.groupBy(subjects)).map((subject) => {
       return ({ title: subject[0], count: subject.length === 1 ?
```

Comment:
count should be **CountLabel**

Reply:
The **input** for this in **cc** is **count**